perception/tracking_object_merger/src/decorative_tracker_merger.cpp

```diff
@@ -220,8 +222,12 @@ void DecorativeTrackerMergerNode::mainObjectsCallback(
 
   // try to merge main object
   this->decorativeMerger(main_sensor_type_, main_objects);
+  const auto & tracked_objects = getTrackedObjects(main_objects->header);
+  merged_object_pub_->publish(tracked_objects);
```

Comment:
```suggestion
```

Why are you publishing these again?

Reply:
If you need a reference to the tracked_objects, you can keep this but you should remove the L228 then.